src/components/FooterPage/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React, {Component} from 'react';
+import { FaInstagram} from 'react-icons/fa';
+import './Footer.scss';
+
+class Footer extends Component {
+    render() {
+        return (
+
+        <div className = "footer_bg pt-5 pb-3  text-center" id = "footer">
+            <div className = "row">
+                <div className = " col-md-6  mr-auto">
+                    <p> <a className = "insta-icon"
+                    href = "https://www.instagram.com/kcelectricalcontractorllc/?igshid=13e62ll0r2ahi"
```

Comment:
I would just clean up the p and a tags format to match the bottom p and a tag layout in this code.
 example==> <p className = "copyright"> ©2021 by <br/>
                    <a href = "https://github.com/ksaini75"> Karishma Saini </a><br/>
                    <a href = "https://github.com/MrGDS2"> MrGDS2 </a>
                    </p>

Reply:
Sure, let me make those changes